### ISSUE TITLE
Home Assistant - Self-signed certificate error

### ISSUE THREAD

**Issue by groypalo** [SynoCommunity/spksrc#4177]
denverton (x64) / DS1618+ / DSM 6.2.3-25426 Update 2 
Home Assistant package v0.114.2-7

**Issue**
When trying to install a package from WebGUI, I get an error. The package is for Sense Home Energy. 
All other packages work fine and the software is stable. Dashboard display all data, etc. for other integrations.

**Some troubleshooting**

```
admin@DS1618:/volume2/@appstore$ pwd
/volume2/@appstore
admin@DS1618:/volume2/@appstore$ openssl s_client -crlf -connect api.sense.com:443 -servername www.sense.com -showcerts
CONNECTED(00000003)
depth=2 C = GB, ST = Greater Manchester, L = Salford, O = COMODO CA Limited, CN = COMODO RSA Certification Authority
verify return:1
depth=1 C = GB, ST = Greater Manchester, L = Salford, O = COMODO CA Limited, CN = COMODO RSA Domain Validation Secure Server CA
verify return:1
depth=0 OU = Domain Control Validated, OU = EssentialSSL Wildcard, CN = *.sense.com
verify return:1
---
Certificate chain
 0 s:/OU=Domain Control Validated/OU=EssentialSSL Wildcard/CN=*.sense.com
   i:/C=GB/ST=Greater Manchester/L=Salford/O=COMODO CA Limited/CN=COMODO RSA Domain Validation Secure Server CA
```

```
admin@DS1618:/volume2/@appstore/homeassistant/bin$ pwd
/volume2/@appstore/homeassistant/bin
admin@DS1618:/volume2/@appstore/homeassistant/bin$ openssl s_client -crlf -connect api.sense.com:443 -servername www.sense.com -showcerts
CONNECTED(00000003)
depth=2 C = GB, ST = Greater Manchester, L = Salford, O = COMODO CA Limited, CN = COMODO RSA Certification Authority
verify return:1
depth=1 C = GB, ST = Greater Manchester, L = Salford, O = COMODO CA Limited, CN = COMODO RSA Domain Validation Secure Server CA
verify return:1
depth=0 OU = Domain Control Validated, OU = EssentialSSL Wildcard, CN = *.sense.com
verify return:1
---
Certificate chain
 0 s:/OU=Domain Control Validated/OU=EssentialSSL Wildcard/CN=*.sense.com
   i:/C=GB/ST=Greater Manchester/L=Salford/O=COMODO CA Limited/CN=COMODO RSA Domain Validation Secure Server CA

```
**running local folder openssl exec**

```
admin@DS1618:/volume2/@appstore/homeassistant/bin$ ./openssl 
OpenSSL> s_client -crlf -connect api.sense.com:443 -servername www.sense.com -showcerts
CONNECTED(00000003)
depth=2 C = GB, ST = Greater Manchester, L = Salford, O = COMODO CA Limited, CN = COMODO RSA Certification Authority
verify error:num=19:self signed certificate in certificate chain
---
Certificate chain
 0 s:/OU=Domain Control Validated/OU=EssentialSSL Wildcard/CN=*.sense.com
   i:/C=GB/ST=Greater Manchester/L=Salford/O=COMODO CA Limited/CN=COMODO RSA Domain Validation Secure Server CA
```

I believe the integration package in the WebGUI for Sense Home Energy monitor leverages the wrong openssl config?
I ran a short script from HomeAssistant SME to troubleshoot initially and it points towards the same root cause : 

```
File "sense.py", line 101, in <module>
asyncio.get_event_loop().run_until_complete(async_test())
File "/volume2/@appstore/python3/lib/python3.7/asyncio/base_events.py", line 587, in run_until_complete
return future.result()
File "sense.py", line 71, in async_test
await sense.authenticate(username, password)
File "/volume2/@appstore/python3/lib/python3.7/site-packages/sense_energy/asyncsenseable.py", line 36, in authenticate
raise SenseAPITimeoutException('Connection failure: %s' % e)
sense_energy.sense_exceptions.SenseAPITimeoutException: Connection failure: Cannot connect to host api.sense.com:443 ssl:True [SSLCertVerificationError: (1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: self signed certificate in certificate chain (_ssl.c:1076)')] 
```

**Comment by BenjV** [comment]
zie:
https://github.com/SynoCommunity/spksrc/issues/4137

**Comment by ymartin59** [comment]
Package updates have been published with fix #4192. Please confirm and close if relevant
